### DISCUSSION TITLE
freeoffice-textmaker: cannot create unique identifier

### DISCUSSION THREAD

**Discussion by hotcapy** [netblue30/firejail#5772] [Q&A] [answered]
Free version of office suite by SoftMaker uses numeric license keys sent by e-mail for app activation.

When launched with Firejail (`firejail freeoffice-textmaker`), app shows the following error:

```
The application cannot create a unique identifier.
Please make sure the application has sufficient permissions.
```

After pressing "Retry" button FreeOffice shows it's usual window requiring to enter free license key.
As you can guess, it's activated already and works w/o Firejail.

There is nothing special in terminal output. Here is how it looks:

![smfo](https://user-images.githubusercontent.com/80588349/229218709-5ea9c234-df62-4a71-889d-30f1c142436d.png)

Perhaps, app stores such activation info somewhere and can't access it. Unfortunately, I don't know where and wasn't able to find out.

I'm using [**freeoffice**](https://aur.archlinux.org/packages/freeoffice) AUR package.

EDIT: Please follow this [issue](https://github.com/netblue30/firejail/issues/5773) instead.

**Comment by kmk3** [comment, marked as answer]
Moved to:

* #5773
